Conan.VisualStudio.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27130.2010
+# Visual Studio Version 16
```

Comment:
why update to 16?

Reply:
Whenever I open this solution with VS2019 it changes that value. Sorry, I didn't realize about it